### DISCUSSION TITLE
User's application screenshots

### DISCUSSION THREAD

**Discussion by platipusica** [jam-py-v5/jam-py#1] [Show and tell]
![jampy01](https://github.com/user-attachments/assets/839d13af-f476-4f79-9ed7-dda3627019e0)

**Comment by crnikaurin** [comment]
![leaves_app](https://github.com/user-attachments/assets/29d1ceed-4501-4e1d-8693-9a38cd8595fd)

Here is calendar for employees leaves tracking

**Reply by 16roi** [reply]
please is it possible to have this work to integrate into my work merci

**Reply by platipusica** [reply]
@crnikaurin this is a question for you :)

**Reply by crnikaurin** [reply]
HI @16roi ofc. You need to have table with employee leaves where to can enter data and create list from that data and fill up fullCalendar with it. I can create some Demo project where you can see how data model is created,

**Reply by 16roi** [reply]
Thank you for your attention. Can I have an example? here's the rest of my project concerning leaves 
![conges](https://github.com/user-attachments/assets/ff4fffb5-ed74-4488-892e-cc5dcec85bd9)

**Comment by crnikaurin** [comment]
![types_of_work](https://github.com/user-attachments/assets/fcbcc63d-bd18-4a09-b575-754089e685b4)

Here is specific report for employees working hours specification...

**Comment by 16roi** [comment]
Here is a work that I started some time ago but my problem is that I can't finish it due to a lack of inspiration and encouragement
![employee](https://github.com/user-attachments/assets/84533453-ac13-4298-862e-608abe15ebdb)
.

**Reply by platipusica** [reply]
I think this example would be perfect for PythonAnywhere with "fake" data if needed. Let us know if we can help.

**Reply by 16roi** [reply]
OK SANS PROBLEME CONTACTEZ MOI

**Reply by platipusica** [reply]
@16roi Hi, not sure how to contact you :(

**Reply by 16roi** [reply]
voila mon email bourahla.omar@gmail.com et merci pour votre attention

**Comment by 16roi** [comment]
voila la suite la puissance de jam.py
![employee2](https://github.com/user-attachments/assets/d5d65455-4674-4ce9-b4cc-f33a207b2f06)

**Comment by platipusica** [comment]
Google authentication (OAuth) - the link will open if user has rights to open it.

![Gdrive](https://github.com/user-attachments/assets/d90aed8a-99d8-43c1-98da-8fe48a261b2f)

**Comment by platipusica** [comment]
Greek Report:
![Manos_LO_24 8 2_go](https://github.com/user-attachments/assets/d2cb8111-e69d-4190-89a1-987d1ff0ae53)
